hack/local-up-karmada.sh

```diff
@@ -126,13 +126,17 @@ kind load docker-image "${REGISTRY}/karmada-scheduler:${VERSION}" --name="${HOST
 kind load docker-image "${REGISTRY}/karmada-descheduler:${VERSION}" --name="${HOST_CLUSTER_NAME}"
 kind load docker-image "${REGISTRY}/karmada-webhook:${VERSION}" --name="${HOST_CLUSTER_NAME}"
 kind load docker-image "${REGISTRY}/karmada-scheduler-estimator:${VERSION}" --name="${HOST_CLUSTER_NAME}"
+kind load docker-image "${REGISTRY}/karmada-metrics-adapter:${VERSION}" --name="${HOST_CLUSTER_NAME}"
 kind load docker-image "${REGISTRY}/karmada-aggregated-apiserver:${VERSION}" --name="${HOST_CLUSTER_NAME}"
 kind load docker-image "${REGISTRY}/karmada-search:${VERSION}" --name="${HOST_CLUSTER_NAME}"
 
 #step5. install karmada control plane components
 "${REPO_ROOT}"/hack/deploy-karmada.sh "${MAIN_KUBECONFIG}" "${HOST_CLUSTER_NAME}"
 
-#step6. wait until the member cluster ready and join member clusters
+#step6. deploy karmada-metrics-adapter
+"${REPO_ROOT}"/hack/deploy-metrics-adapter.sh "${MAIN_KUBECONFIG}" "${HOST_CLUSTER_NAME}"
```

Comment:
Do we need to install metrics-adapter by default when running `hack/local-up-karmada.sh`? cc @XiShanYongYe-Chang for checking

Reply:
Can we just provide the script and let the user decide whether to install it?

Reply:
+1

Reply:
Should we remove it then?

Reply:
I prefer to delete it

Reply:
Sure!